services/jvm/src/main/scala/io/gearpump/services/MasterService.scala

```diff
@@ -201,10 +201,10 @@ object MasterService {
   /**
    * Submit Native Application.
    */
-  def submitGearApp(jar: Option[File], args: String, systemConfig: Config, userConfigFile: Option[File]): Boolean = {
+  def submitGearApp(jar: Option[File], executorNum: String, args: String, systemConfig: Config, userConfigFile: Option[File]): Boolean = {
```

Comment:
It is not safe to use String, as the executorNum is user provided value from http request, we need to sanitize it.